stable/heapster/templates/deployment.yaml

```diff
@@ -30,6 +30,10 @@ spec:
           timeoutSeconds: 5
         resources:
 {{ toYaml .Values.resources | indent 12 }}
+{{- if .Values.env }}
+        env:
+{{ toYaml .Values.env | indent 10 }}
```

Comment:
Because `env` only requires key-value pairs, our pattern is:
```
          env:
          {{- range $key, $value := .Values.env }}
            - name: {{ $key }}
              value: {{ $value }}
          {{- end }}
```
As @gyliu513 mentioned, be sure to add `env : []` to `values.yaml`.

Reply:
@mgoodness hmm users should be able to pass ['valueFrom'](https://kubernetes.io/docs/api-reference/v1.7/#envvar-v1-core) field as well. So hash of string value won't work.